spec/money_spec.rb

```diff
@@ -851,12 +857,8 @@ def m.amount
   end
 
   describe ".default_currency" do
-    before do
-      @default_currency = Money.default_currency
-    end
-
     after do
-      Money.default_currency = @default_currency
+      Money.default_currency = nil
```

Comment:
This change makes the test coupled to the default value, previous solution however does not, which is definitely better